src/lib.rs

```diff
@@ -24,23 +31,40 @@ pub fn App() -> impl IntoView {
     <Meta name="viewport" content="viewport-fit=cover"/>
     <Script src="//cdn.jsdelivr.net/npm/eruda"/>
     <Script>eruda.init();</Script>
+    <Title text="Brand from env"/>
 
     // adding `set_is_routing` causes the router to wait for async data to load on new pages
     <Router set_is_routing>
-      // shows a progress bar while async data are loading
-      <RoutingProgress is_routing max_time=std::time::Duration::from_millis(250)/>
-      <TopNav/>
-      <main>
-        <Routes>
-          <Route path="home" view=HomeActivity/>
-          <Route path="" view=HomeActivity/>
-          <Route path="login" view=LoginActivity/>
-          <Route path="post/:id" view=PostActivity/>
-        // <Route path="stories/:id" view=Story/>
-        // <Route path=":stories?" view=Stories/>
-        </Routes>
-      </main>
-      <BottomNav/>
+      <div class="flex flex-col h-screen">
+        // shows a progress bar while async data are loading
+        <RoutingProgress is_routing max_time=std::time::Duration::from_millis(250)/>
+        <TopNav/>
+        <main class="container mx-auto">
```

Comment:
I think it'd be better to let each Activity set its own container, and be fully configurable. Some might want to do some full-width things, who knows.

Reply:
ok